src/builder.rs

```diff
@@ -344,6 +346,18 @@ impl Builder {
         Ok(())
     }
 
+    /// Add an instruction to burn given amount os specific ZSA
```

Comment:
`Add an instruction to burn a given amount of a specific asset.`

Reply:
done